execution/scenarioExecutor.go

```diff
@@ -145,12 +145,18 @@ func (e *scenarioExecutor) notifyBeforeConcept(conceptResult *result.ScenarioRes
 	message := &gauge_messages.Message{MessageType: gauge_messages.Message_ConceptExecutionStarting,
 		ConceptExecutionStartingRequest: &gauge_messages.ConceptExecutionStartingRequest{CurrentExecutionInfo: e.currentExecutionInfo, Stream: int32(e.stream)}}
 	e.pluginHandler.NotifyPlugins(message)
+	if (e.runner.Info().ConceptMessages) {
+		_ = e.runner.ExecuteAndGetStatus(message)
```

Comment:
The only feedback I have here (and I apologize for dragging this) is that these ConceptHooks are not sending any result back to the runner. As a result there will be no failure reported, no messages or screenshots will be accumulated.

This is perhaps obvious, but I want to make sure that you are aware of this.

Reply:
We're aware that presently there is no response sent to the gauge runner. An exception in the Concept hooks results only in a (Debug) log output.

It is possible to update gauge-proto NotifyConceptExecutionStarting NotifyConceptExecutionEnding with ExecutionStatusResponse and deliver the proper response.
The question is then how Reporters will handle errors on this Concept step level?

**We suggest to merge this PR. 
We propose to investigate and solve the error management in a subsequent implementation.**